editor/components/block-list/style.scss

```diff
@@ -210,7 +209,8 @@
 	/**
 	 * Block outline layout
 	 */
-	.editor-block-list__block-edit {
+
+	 .editor-block-list__block-edit {
```

Comment:
Mixed whitespace. Preceding space should be removed.

Reply:
Nice catch, thank you.